doc/development/rfc/release_policy.md

```diff
@@ -0,0 +1,118 @@
+# RFC: Release Policy
+
+Author: Vaclav Petras
+
+Status: Draft
+
+## Goal
+
+The goal of this document is to define when releases should be created
+and what types releases should be created.
+
+## Summary
+
+One major or minor release should be released once a year.
+This becomes the only actively maintained release series
+which is subsequently updated by micro releases as needed.
+Individual contributors can step up to maintain an older release series.
+
+## Types of Releases
+
+We recognize major, minor, micro releases and additional helper versions
+each with its unique purpose.
+
+The releases follow rules and numbering specified in the Version Numbering RFC.
+
+### Major and Minor
+
+Both major and minor releases bring new features to the users
+and in that regard should receive similar attention by the contributors.
+However, they differ in how backwards compatibility is handled.
+
+While major release brings features just like a minor release,
+it may also bring changes breaking backwards compatibility
+such as breaking API changes. Additionally, contributors may decide
+to include certain feature into a major release rather than a minor release
+if the feature has a potential to negatively impact some users or use cases.
+A major release may include a particularly important feature or a set of features,
+but that's not required. The purpose of a major release is to bring changes
+which cannot be included into a minor release such as A or some extensive code rewrites.
+It may or may not be used to bring revolutionary changes or highlight specific features.
+
+Minor release may bring any new features, small or big, as long as backwards
+compatibility is kept, e.g., an automated procedure relying on public API should
+still work. The purpose of a minor release is to bring features to the users
+as soon as possible with software stability and compatibility and without
+a need to wait for a major release.
+
+Both major and minor releases may include bugfixes which may or may not be
+already included in a micro release of the previous major or minor release.
+
+A special effort is put into keeping backwards compatibility.
+While this is a requirement for minor version, the community makes extra effort
+even for major release to maximize backwards compatibility so that users can
```

Comment:
```suggestion
While this is a requirement for minor versions, the community makes extra effort
even for major releases to maximize backwards compatibility so that users can
```